src/utils/queryOps.js

```diff
@@ -21,92 +21,91 @@ class Wrapper {
   fromPairs() {
     return lodashFromPairs(this.list);
   }
-
 }
 
-const toPairs = (obj) => {
-  return new Wrapper(lodashToPairs(obj));
-};
-
-const frum = (list) => {
-  return new Wrapper(list);
-};
-
-
+const toPairs = obj => new Wrapper(lodashToPairs(obj));
+const frum = list => new Wrapper(list);
 // Add a chainable method to Wrapper
-const extend_wrapper = (methods) => {
+const extendWrapper = methods => {
   lodashToPairs(methods).forEach(([name, method]) => {
     // USE function(){} DECLARATION TO BIND this AT CALL TIME
     Wrapper.prototype[name] = function anonymous(...args) {
       this.list = method(this.list, ...args);
+
       return this;
     };
   });
 };
 
-
 // Add Lodash functions
-extend_wrapper({
-  map: map,
-  lodashFilter: lodashFilter,
-  flatten: flatten,
+extendWrapper({
+  map,
+  lodashFilter,
+  flatten,
   toPairs: lodashToPairs,
-  chunk: chunk,
-  unzip: unzip,
-  sortBy: sortBy,
-  lodashGroupBy: lodashGroupBy,
+  chunk,
+  unzip,
+  sortBy,
+  lodashGroupBy,
   limit: lodashTake,
 });
 
-
-const toArray = (value) => {
+const toArray = value => {
   if (Array.isArray(value)) {
     return value;
-  } if (value == null) {
+  }
+
+  if (value == null) {
     return [];
   }
-    return [value];
+
+  return [value];
 };
 
 // convert string into function that selects column from row
-const selector = (column_name) => {
-  if (typeof column_name === 'string') {
-    return row => row[column_name];
+const selector = columnName => {
+  if (typeof columnName === 'string') {
+    return row => row[columnName];
   }
-    return column_name;
 
+  return columnName;
 };
 
-
-extend_wrapper({
+extendWrapper({
   // Groupby one, or many, columns by name or by {name: selector} pairs
   // return array of [rows, key, index] tuples
   groupBy: function groupBy(list, columns) {
     const cs = frum(toArray(columns))
-      .map((col_name) => {
-        if (typeof col_name === 'string') {
-          return [[col_name, selector(col_name)]];
+      .map(colName => {
+        if (typeof colName === 'string') {
+          return [[colName, selector(colName)]];
         }
-        return lodashToPairs(col_name).map(([name, value]) => [name, selector(value)]);
+
+        return lodashToPairs(colName).map(([name, value]) => [
+          name,
+          selector(value),
+        ]);
       })
       .flatten()
-      .sortBy(([col_name]) => col_name)
+      .sortBy(([colName]) => colName)
```

Comment:
@aimenbatool  Was this name change automatic, or did you do this manually?

Reply:
This was automatic I think, for many, I disabled this rule as it was giving linting issue.  
If this is working fine then should we keep it as is?

Reply:
Nice!  Automatic is nice!

Reply:
:) Just looked into my notes about what I manually changed, I changed all variables from underscore to camelCase :) 
This isn't provided automatically. 
Actually, I'm not sure what things were done automatically because I did following manually:
- variable name convention
- single quote to double quote
- indentation 
- changing files from js to.jsx
- function fixing

I think trailing commas and a few things are fixed automatically.